### PR TITLE
CI: increase Docker compression level

### PR DESCRIPTION
**Pull request by pjonsson**
Increasing the compression level from 19
to 20 gives significant space savings
without increasing docker load times.
Increase the compression level to the max
since that will benefit every download
performed by the CI.

Compression level is documented, but it
does currently not have any effect
for the docker exporter. If level 22
turns out to take prohibitively long
when the parameter is no longer ignored,
this setting can be dialed back to 20.

### CODE REVIEW

**Comment by pjonsson** [comment]
There is no size difference from the increased compression level. Investigating.

**Comment by pjonsson** [comment]
There is no difference between compression-level 12 and 22.

Changing parameter order, or removing force-compression, does not make any difference.

Putting this as a draft for now.

**Comment by pjonsson** [comment]
My mental model was that we wait until the Docker bug is fixed, and then merge this (since we want faster CI runs). I don't want to keep track of what is fixed for every release of the entire Docker software stack, so I think it's better to merge this now and get a pleasant surprise when compression-level is no longer ignored by Docker.

There is a risk that level 22 takes too long, but it's easy to dial it back to level 20 if that happens. It's also not a subtle thing, the image builder jobs taking forever is something we will notice, and the CI logs make it clear that the image is built and is in the process of being exported/compressed.

**Review by keith-packard — APPROVED** [review]

**Pull request merged**
